docs/src/components/docs/SideNav.js

```diff
@@ -0,0 +1,153 @@
+import * as React from "react"
+import { sortBy, upperFirst, camelCase } from "lodash/fp"
+import { graphql, useStaticQuery } from "gatsby"
+import { Box, Heading } from "@chakra-ui/core"
+import { ComponentLink, TopNavLink } from "./NavLink"
+
+const sortByOrder = sortBy(["frontmatter.order"])
+const sortByTitle = sortBy(["frontmatter.title"])
+
+const useLinksQuery = () => {
```

Comment:
We use this to query for each individual links collection: `main`, `components`, and `utilities`. The resulting object has the `nodes` for each under the same key.